usr/lib/common/aix/err.h

```diff
@@ -0,0 +1,26 @@
+#if defined(__WIN32) || defined(_AIX)
```

Comment:
Where does this implementation come from? If you wrote it yourself, then this file needs a copyright and license header, otherwise we should include the original header....